SpatialGDK/Documentation/content/get-started/starter-template/get-started-template-intro.md

```diff
@@ -23,7 +23,7 @@ We recommend setting up the [Example Project]({{urlRoot}}/content/get-started/ex
 
 </br>
 
- ### **> Next:** [1. Set up]({{urlRoot}}/content/get-started/starter-template/get-started-template-setup)
+#### **> Next:** [1. Set up]({{urlRoot}}/content/get-started/starter-template/get-started-template-setup)
```

Comment:
the currently live page has broken formatting for this which we should make sure is fixed by this change
![2019-08-21 13_42_26-Get started - SpatialOS SDK Documentation](https://user-images.githubusercontent.com/2433131/63432721-8d93a200-c419-11e9-8bcb-d0bef1ff3051.png)
https://docs.improbable.io/unreal/alpha/content/get-started/starter-template/get-started-template-intro

Reply:
All fixed on staging: https://docs-staging.improbable.io/unreal/alpha/content/get-started/starter-template/get-started-template-intro